### ISSUE TITLE
Reproducing baseline

### ISSUE THREAD

**Issue by haoheliu** [c4dm/dcase-few-shot-bioacoustic#18]
@shubhr Hi! Can you please share some info on training the baseline model? I experience some difficulties to reproduce. It would be helpful to know what's the value of training loss/acc when the model converges.

**Comment by shubhr** [comment]
Hi. The training accuracy at epoch 49 was 98.6%. I have updated the config.yaml and the main.py. Can you check if you are facing the same issues in training

**Comment by SarthakJShetty** [comment]
Also, it seems like the root_dir in the config.yaml don't match the structure of the dataset provided on the website? For example this is the ```root.dir``` in the [```config.yaml```](https://github.com/c4dm/dcase-few-shot-bioacoustic/blob/main/baselines/deep_learning/config.yaml):

```
path:
  root_dir:  /import/research_c4dm/ss380/Bird_Dev_train
```

But the dataset released for the Bioacoustic Challenge has the following structure:

```
user@machine: ~/dcase-few-shot-bioacoustic/baselines/deep_learning/dataset$ ls
Development_Set  Development_Set_Annotations  Development_Set_Annotations.zip  Development_Set_Audio  Development_Set_Audio.zip  Development_Set.zip  README.md
```

Is there some other file structured as ```research_c4dm``` that I should be looking at?

Thanks!

**Comment by shubhr** [comment]
HI Sarthak, the root directory is dependant on where you want to store your model weights and features. Its dependant on user. For the training data, the train_dir parameter is what sets the path to the training data.

**Comment by SarthakJShetty** [comment]
Thanks @shubhr. I got the baseline training!

**Comment by liushenme** [comment]
Hi, I have the same problem that it is hard to get the reported results when retraining the deep learning based-baseline. I updated the config.yaml and the main.py and the the training accuracy at epoch 49 was only 63.9%. The final reuslts I got  is 1.81%. Why I got this poor results? 
![a290d5e79b94cb25d981d21af08488d](https://user-images.githubusercontent.com/49808596/162892818-f7d21fef-c01d-4b86-b0f7-a5efabc18286.png)

**Comment by Noumanijaz744** [comment]
hi everyone i have done feature extraction, training and evaluation of the model through config.yaml set.eval:true .but i want to create the evaluation metrics how it can be created .can you please guide?

**Comment by chester-w-xie** [comment]
Evaluation for: TESTteam VAL
check 1 : ['ME', 'HB', 'PB']
BUK1_20181011_001004.wav {'TP': 2, 'FP': 755, 'FN': 29, 'total_n_pos_events': 31}
BUK1_20181013_023504.wav {'TP': 0, 'FP': 576, 'FN': 24, 'total_n_pos_events': 24}
BUK4_20161011_000804.wav {'TP': 0, 'FP': 25, 'FN': 47, 'total_n_pos_events': 47}
BUK4_20171022_004304a.wav {'TP': 7, 'FP': 80, 'FN': 10, 'total_n_pos_events': 17}
BUK5_20161101_002104a.wav {'TP': 0, 'FP': 1, 'FN': 88, 'total_n_pos_events': 88}
BUK5_20180921_015906a.wav {'TP': 1, 'FP': 1118, 'FN': 22, 'total_n_pos_events': 23}
ME1.wav {'TP': 5, 'FP': 85, 'FN': 6, 'total_n_pos_events': 11}
ME2.wav {'TP': 16, 'FP': 39, 'FN': 25, 'total_n_pos_events': 41}
R4_cleaned recording_13-10-17.wav {'TP': 16, 'FP': 18, 'FN': 3, 'total_n_pos_events': 19}
R4_cleaned recording_16-10-17.wav {'TP': 21, 'FP': 17, 'FN': 9, 'total_n_pos_events': 30}
R4_cleaned recording_17-10-17.wav {'TP': 5, 'FP': 46, 'FN': 31, 'total_n_pos_events': 36}
R4_cleaned recording_TEL_19-10-17.wav {'TP': 46, 'FP': 13, 'FN': 8, 'total_n_pos_events': 54}
R4_cleaned recording_TEL_20-10-17.wav {'TP': 62, 'FP': 14, 'FN': 2, 'total_n_pos_events': 64}
R4_cleaned recording_TEL_23-10-17.wav {'TP': 76, 'FP': 4, 'FN': 8, 'total_n_pos_events': 84}
R4_cleaned recording_TEL_24-10-17.wav {'TP': 65, 'FP': 23, 'FN': 34, 'total_n_pos_events': 99}
R4_cleaned recording_TEL_25-10-17.wav {'TP': 67, 'FP': 58, 'FN': 32, 'total_n_pos_events': 99}
file_423_487.wav {'TP': 52, 'FP': 22, 'FN': 5, 'total_n_pos_events': 57}
file_97_113.wav {'TP': 13, 'FP': 44, 'FN': 107, 'total_n_pos_events': 120}

Overall_scores: {'precision': 0.011320026175091482, 'recall': 0.11094181268655784, 'fmeasure (percentage)': 2.054}
